scripts/docs_config/conf.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2023-2024 Intel Corporation
```

Comment:
I guess it's redundant in this file, it's just a configuration file, not a source code